packages/dialtone-vue2/components/split_button/split_button.vue

```diff
@@ -0,0 +1,282 @@
+            v-bind="{ ...attrs, ...omegaButtonProps }"
+            @click="(e) => $emit('omega-clicked', e)"
+          >
+            <template #icon>
+              <!-- @slot Omega (right) button icon slot -->
+              <slot name="omegaIcon" />
+            </template>
+          </split-button-omega>
+        </template>
+        <template #list>
+          <!-- @slot Built-in dropdown contents -->
+          <slot name="dropdownList" />
+        </template>
+      </dt-dropdown>
+
+      <split-button-omega
+        v-else
+        v-bind="omegaButtonProps"
+        @click="(e) => $emit('omega-clicked', e)"
+      >
+        <template #icon>
+          <!-- @slot Omega (right) button icon slot -->
+          <slot name="omegaIcon" />
+        </template>
+      </split-button-omega>
+    </slot>
+  </span>
+</template>
+
+<script>
+import {
+  DtButton,
+  BUTTON_IMPORTANCE_MODIFIERS,
+  BUTTON_KIND_MODIFIERS,
+  BUTTON_SIZE_MODIFIERS,
+  ICON_POSITION_MODIFIERS,
+} from '@/components/button';
+import SplitButtonOmega from './split_button-omega.vue';
+import { DtDropdown } from '@/components/dropdown';
+
+export default {
+  name: 'DtSplitButton',
+
+  components: {
+    DtButton,
+    SplitButtonOmega,
+    DtDropdown,
+  },
+
+  props: {
+    /**
+     * Determines whether the alpha button should have active styling
+     * default is false.
+     * @values true, false
+     */
+    alphaActive: {
+      type: Boolean,
+      default: false,
+    },
+
+    /**
+     * Descriptive label for the alpha button
+     */
+    alphaAriaLabel: {
+      type: String,
+      default: null,
+    },
+
+    /**
+     * The position of the icon slot within the alpha button.
+     * @values left, right, top, bottom
+     */
+    alphaIconPosition: {
+      type: String,
+      default: 'left',
+      validator: (position) => Object.keys(ICON_POSITION_MODIFIERS).includes(position),
+    },
+
+    /**
+     * Used to customize the alpha label container
+     */
+    alphaLabelClass: {
+      type: [String, Array, Object],
+      default: '',
+    },
+
+    /**
+     * Whether the alpha button should display a loading animation or not.
+     * @values true, false
+     */
+    alphaLoading: {
+      type: Boolean,
+      default: false,
+    },
+
+    /**
+     * Text shown in tooltip when you hover the alpha button
+     */
+    alphaTooltipText: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * Determines whether a screenreader reads live updates of
+     * the button content to the user while the button
+     * is in focus. default is to not.
+     * @values true, false
+     */
+    assertiveOnFocus: {
+      type: Boolean,
+      default: false,
+    },
+
+    /**
+     * HTML button disabled attribute
+     * <a
+     *   class="d-link"
+     *   href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#disabled"
+     *   target="_blank"
+     * >
+     *   (Reference)
+     * </a>
```

Comment:
Here too, not sure it is necessary to have a fully explanation for a `disable` attribute..

Reply:
It's not a full explanation, is just a link to help in case you need more information about it. There are a lot of details about disabled attribute that no everyone knows.